.mergify.yml

```diff
@@ -32,3 +32,10 @@ pull_request_rules:
     actions:
       delete_head_branch:
         force: true
+  - name: Block regular PRs from merging into master
+    conditions:
+      - base=master
+      - head~=^(?!release.*).*$
+    actions:
+      close:
+        message: This repository is using GitFlow. New features should be merged into `dev` instead of master. Please open a new pull request that targets `dev` instead.
```

Comment:
```suggestion
        message: This repository is using GitFlow. New features should be merged into `dev` instead of master. Please open a new pull request that targets `dev` instead.

```

Reply:
Had to reconfigure my editor, thanks :)